app/services/can_recruit_with_pending_conditions.rb

```diff
@@ -0,0 +1,50 @@
+class CanRecruitWithPendingConditions
+  attr_accessor :application_choice
+
+  def initialize(application_choice:)
+    self.application_choice = application_choice
+  end
+
+  def call
+    feature_flag_enabled? &&
+      application_has_offer? &&
+      offer_has_pending_ske_conditions? &&
+      offer_has_no_unmet_non_ske_conditions? &&
```

Comment:
this triple-negative name is a bit of a brainteaser. how about `all_non_ske_conditions_met?`

Reply:
fair point!